docs/lagoon-projects.md

```diff
@@ -10,12 +10,14 @@ Lagoon](./runbooks/add-generic-site-to-platform/). These usually
 automatically build select branches and pull requests and are
 primarily for testing in development.
 
-Currently limited to the main `dpl-cms`.
+Currently this includes `dpl-cms` project (main development) and the
+`dpl-bnf` project ("Bibliotekernes Nationale Formidling" content
+sharing system).
 
 ## Production/staging sites
 
-Production and staging sites are [set up in
-`sites.yml`](./runbooks/add-library-site-to-platform/). This process
+Sites managed by
```

Comment:
This change does not make sense to me, what's your reasoning here?

Reply:
Just a bit of rephrasing. Starting a paragraph with "Production and staging sites" right after a "Production/staging sites" headline is redundant.